tests/test_custom_checker.py

```diff
@@ -1,10 +1,10 @@
 from xmltodict import parse
-
+import sys
+import os
+sys.path.append(os.getcwd())
 from pyQuARC.code.custom_checker import CustomChecker
-
-from .fixtures.custom_checker import INPUT_OUTPUT
-from .common import read_test_metadata
-
+from tests.fixtures.custom_checker import INPUT_OUTPUT
+from tests.common import read_test_metadata
```

Comment:
were the tests not working without?

Reply:
Receive the following error when running tests/test_custom_checker.py with
```
from xmltodict import parse
from pyQuARC.code.custom_checker import CustomChecker

```
```
Traceback (most recent call last):
  File "/Users/sam/Documents/GitHub/pyQuARC/tests/test_custom_checker.py", line 2, in <module>
    from pyQuARC.code.custom_checker import CustomChecker
ModuleNotFoundError: No module named 'pyQuARC'
```